ui-speedspacechart/src/components/SpeedSpaceChart.tsx

```diff
@@ -90,6 +90,7 @@ const SpeedSpaceChart = ({
   const dynamicHeightOffset = getAdaptiveHeight(HEIGHT_OFFSET, store.layersDisplay);
 
   const [showDetailsBox, setShowDetailsBox] = useState(false);
+  const [mouseIsOverSettingsPanel, setMouseIsOverSettingsPanel] = useState(true);
```

Comment:
To match the usual css vocabulary, shouldn't we use "hover" for this state ?

```suggestion
  const [mouseHoveringSettingsPanel, setMouseHoveringSettingsPanel] = useState(true);
```

Reply:
you're right if we talk about css, but here it's JSX rendering, anyway if you want this change I can do it

Reply:
to be more explicit: if this state triggers a class I can accept it's about css, but it isn't what it does

Reply:
I think if the mouse is hovering over the panel, the mouse being **over** the panel is the important part.